### DISCUSSION TITLE
OCaml color change

### DISCUSSION THREAD

**Discussion by Teddy55Codes** [github-linguist/linguist#6291] [General]
Here you can voice your opinion about the color change from `#3be133` to `#ee6a1a`.
This change and the reasons for it are further described in the pull request [#6284](https://github.com/github/linguist/pull/6284).

**Comment by lildude** [comment]
Please don't. These discussions are for discussing Linguist issues. Any community discussions for colour changes should take place on that language's official channels. From the CONTRIBUTING.md:

> If you would like to change the color of a language, we ask that you propose your suggested color change to the wider community for your language to gain consensus before submitting a pull request. Please do this in a community forum or repository used and known by the wider community of that language, not the Linguist repository.

Accordingly, I'm going to lock the discussion. Please update the OP of https://github.com/github/linguist/pull/6284 with the public OCaml community location this discussion is taking place.
